how-to-guides/consensus-node.md

```diff
@@ -342,12 +342,59 @@ Optional: If you would like celestia-app to run as a background process, you can
 
 ### Optional: Reset network
 
-This will delete all data folders so we can start fresh:
+There are several ways to reset your consensus node, depending on what you need to preserve:
+
+#### Option 1: Reset blockchain data while preserving configuration
+
+This command removes blockchain data but keeps your node configuration:
+
+```sh
+celestia-appd tendermint reset-state
+```
+
+This preserves your configuration but removes:
+- blockstore.db
+- state.db
+- evidence.db
+
+#### Option 2: Manual reset (recommended for validator nodes)
+
+For more precise control over what gets reset:
+
+```sh
+mv ~/.celestia-app/data ~/.celestia-app/old-data
+mkdir -pv ~/.celestia-app/data
+cp ~/.celestia-app/old-data/priv_validator_state.json ~/.celestia-app/data/priv_validator_state.json
+```
+
+This approach lets you selectively preserve critical validator state files.
```

Comment:
```suggestion
This approach allows you to selectively preserve important validator state files. It restores only the validator state, not the validator key. By default, the validator key is stored in `priv_validator_key.json`, located at `~/.celestia-app/config/priv_validator_key.json`, so it will remain unchanged.
```